src/main/java/com/naijagov/naijagov/service/PollingUnitService.java

```diff
@@ -0,0 +1,29 @@
+package com.naijagov.naijagov.service;
+
+import java.util.Optional;
+
+import com.naijagov.naijagov.model.PollingUnit;
+import com.naijagov.naijagov.repository.PollingUnitRepository;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service("pollingUnitService")
+public class PollingUnitService{
```

Comment:
Same observation as before

Reply:
Please refer to [previous response](https://github.com/bioye/naijagov/pull/39#discussion_r294046126) above.